webapp/packages/core-blocks/src/ObjectPropertyInfo/ObjectPropertyInfoForm/RenderField.tsx

```diff
@@ -55,25 +63,42 @@ function getValue(value: any, controlType: ObjectPropertyType) {
 export const RenderField = observer<RenderFieldProps>(function RenderField({
   property,
   state,
+  context = state,
   defaultState,
   editable = true,
   autofillToken = '',
   disabled,
-  readOnly,
   autoHide,
   showRememberTip,
   saved,
   className,
   canShowPassword,
   onFocus,
+  ...rest
 }) {
   const translate = useTranslate();
 
+  let readonly = rest.readOnly;
```

Comment:
It is a strange move with read-only. Let's revert it as it was from props as `readOnly`, then you can change it if needed (it's not props mutation)